Source/PINDiskCache.m

```diff
@@ -1453,4 +1525,10 @@ - (void)enumerateObjectsWithBlock:(PINDiskCacheFileURLBlock)block completionBloc
 @end
 
 @implementation PINDiskCacheMetadata
+
+- (void)setAgeLimit:(NSTimeInterval)ageLimit {
+    _ageLimit = ageLimit;
+    _hasAgeLimit = YES;
```

Comment:
Would it be better to just make a getter for `hasAgeLimit` which checks `_ageLimit > 0`?

Reply:
My thoughts were if `ageLimit` were set to anything (including zero), it would override the cache's `ageLimit`. I realize this is actually confusing, so I removed `hasAgeLimit`; the object's `ageLimit` must now be greater than zero in order to override the cache's `ageLimit`.